src/main/scala/beam/utils/ConfigValuesComparator.scala

```diff
@@ -0,0 +1,55 @@
+package beam.utils
+
+import com.typesafe.config.ConfigRenderOptions
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.collection.JavaConverters._
+
+object ConfigValuesComparator extends LazyLogging {
+
+  def getFullConfigMap(configFileLocation: String): Predef.Map[String, String] = {
+    val renderOptions = ConfigRenderOptions.concise()
+
+    val config = BeamConfigUtils.parseFileSubstitutingInputDirectory(configFileLocation)
+    config
+      .entrySet()
+      .asScala
+      .map(x => x.getKey -> x.getValue.render(renderOptions).split(',').last)
```

Comment:
Does this handle nested configurations? I'd be especially curious about nested configs that are mismatched on order.

Reply:
It will handle nested configurations and if the order of included configs is different and included configs contains the same key, it will use the latest version of the value.

Reply:
I think the newest change is what does that now. thanks!